kernel-crawler/scripts/run-crawler.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
Do we absolutely need to drive the crawlers through a python script? Seems like overkill to me.

Reply:
The primary reason I picked python was to make the argument parsing clearer and more robust. With bash, it'd be much more difficult to group lists of arguments together (e.g for `-e` and `-v`) 

Appreciate what you're saying though and if there's an easy way of achieving this in bash or something else, I'm all ears

Reply:
Seems to be a good idea to me, especially since it does `execvp`, without messing around with subprocess or such.